internal/services/ch/ch.go

```diff
@@ -0,0 +1,101 @@
+package ch
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"time"
+
+	"github.com/ClickHouse/clickhouse-go/v2"
+	"github.com/DIMO-Network/rewards-api/internal/config"
+)
+
+const (
+	defaultMaxExecutionTime                    = 5
```

Comment:
This seems too short. It my testing this query took at least 20 seconds. `source` isn't part of the index so it takes a while.

Like I would set this to be something really large. We need this query to succeed or die trying.

Reply:
I am guessing you just copied this from the telemetry API, which needs this setting for its specific use case.
We can use a basic client setup without a custom execution time
```golang
import "github.com/DIMO-Network/clickhouse-infra/pkg/connect"
func NewClient(settings *config.Settings) (*Client, error) {
	chConn, err := connect.GetClickhouseConn(&settings.Clickhouse)
	if err != nil {
		return nil, fmt.Errorf("failed to create ClickHouse connection: %w", err)
	}
	...
```